src/schemathesis/filters.py

```diff
@@ -26,3 +26,10 @@ def should_skip_by_tag(tags: Optional[List[str]], pattern: Optional[Filter]) ->
         return True
     patterns = force_tuple(pattern)
     return not any(re.search(item, tag) for item in patterns for tag in tags)
+
+
+def should_skip_by_operation_id(operation_id: str, pattern: Optional[Filter]) -> bool:
+    if pattern is None:
+        return False
+    patterns = force_tuple(pattern)
+    return operation_id not in patterns
```

Comment:
probably we can also support regular expressions here, which will be the same implementation as in `should_skip_endpoint`

Reply:
good idea! I haven't seen anything about Re in the documentation before, but this is really an exciting feature, maybe you can update the documentation? : P

Reply:
I'll add it to our todo list! Thank you for the suggestion. #500